features/fixtures/sampler/ios/sampler.xcodeproj/project.pbxproj

```diff
@@ -40,6 +39,8 @@
 		832341BD1AAA6AB300B99B32 /* libRCTText.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 832341B51AAA6A8300B99B32 /* libRCTText.a */; };
 		ADBDB9381DFEBF1600ED6528 /* libRCTBlob.a in Frameworks */ = {isa = PBXBuildFile; fileRef = ADBDB9271DFEBF0700ED6528 /* libRCTBlob.a */; };
 		F7023905233C48118ADFE9D6 /* libz.tbd in Frameworks */ = {isa = PBXBuildFile; fileRef = 41820A385C704024802F33EE /* libz.tbd */; };
+		16C36C00A7E041AAB76EF567 /* libReactNativeConfig.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 873456FFA9624C3CA83A37C3 /* libReactNativeConfig.a */; };
```

Comment:
Why are there ios changes when this PR adds android tests?

Reply:
This is due to the linking of the React-Native-Config module, as mentioned below.  I think it's worth linking it to both flavours, as if we come to use it again it'd be strange that it's only linked to one.